test/e2e/ginkgo/multiple_cluster_deployment_test.go

```diff
@@ -514,6 +515,36 @@ var _ = ginkgo.Describe("end to end testing", func() {
 			gomega.Expect(*cs.Results[0].Rows[0].Columns[0].Number).To(gomega.Equal("2"))
 		})
 
+		ginkgo.It("list of WECs with available nginx deployment", func(ctx context.Context) {
```

Comment:
I think that it would be better to be a little more specific in the description of the test here, perhaps something like "lists all the WECs where the reported number of availableReplicas equals the desired number of replicas".

Reply:
If we keep the label check from the comment above than we need to say `nginx deployment` in the test name.

Reply:
Yes. But remember that `ginkgo.It` takes not a name but an English sentence (with the subject elided) asserting the behavior that is to be tested.

Reply:
Changed.